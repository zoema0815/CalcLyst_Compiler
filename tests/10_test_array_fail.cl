int main () {
    int[2] int_list;
    int_list[2] = 12;
    return 0;
}