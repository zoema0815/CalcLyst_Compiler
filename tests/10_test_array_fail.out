Fatal error: exception Failure("ERROR: Index out of bounds.")
